Actions/DetermineProjectsToBuild/DetermineProjectsToBuild.ps1

```diff
@@ -50,39 +50,58 @@ function Get-FilteredProjectsToBuild($settings, $projects, $baseFolder, $modifie
     if ($settings.alwaysBuildAllProjects) {
         Write-Host "Building all projects because alwaysBuildAllProjects is set to true"
         return $projects
-    } 
+    }
 
     if (!$modifiedFiles) {
         Write-Host "No files modified, building all projects"
         return $projects
     }
 
-    if ($modifiedFiles -like '.github/*.json') {
-        Write-Host "Changes to repo Settings, building all projects"
-        return $projects
-    }
-    
+    Write-Host "$($modifiedFiles.Count) modified file(s): $($modifiedFiles -join ', ')"
+
     if ($modifiedFiles.Count -ge 250) {
         Write-Host "More than 250 files modified, building all projects"
         return $projects
     }
 
-    Write-Host "$($modifiedFiles.Count) modified file(s): $($modifiedFiles -join ', ')"
+    $fullBuildPatterns = @( Join-Path '.github' '*.json')
```

Comment:
Could this be the added as the default value for the fullBuildPatterns setting instead? I think if you decide to use this setting it should use only the build patterns you specify

Reply:
IMO The problem of adding this as a default would be that people would have to remember to include it if they create the setting - I think the current mechanism is correct - we have some settings, which people cannot remove.

Reply:
A little documentation on the setting would reduce that risk :) I just don't love how we inject settings like this or have settings that people cannot remove. 

Anyway.. It's not something I feel strong enough about to block the PR. It's up to you Maria. Feel free to dismiss my comment if you disagree.

Reply:
My thinking in this case was what Freddy described: it's very easy to forget to add `.github/*.json` and this is a _mandatory_ check (as changing the AL-Go settings should re-build all projects).

One other mandatory file check would be _PullRequestHandler.yaml_ but only of the trigger is `pull_request`.